packages/client/src/i18n/messages/views/userSetup.ts

```diff
@@ -356,10 +358,23 @@ const messagesToDefine: IUserSetupMessages = {
     description: 'Description for User reset password action',
     id: 'user.profile.auditList.passwordReset'
   },
+
+  sendPasswordAuditAction: {
+    defaultMessage: 'Sent Password',
+    description: 'Description for sending temporal password action',
+    id: 'user.profile.auditList.sentPassword'
+  },
+
   userNameReminderAuditAction: {
     defaultMessage: 'Username Reminder Requested',
     description: 'Description for User username requested audit action',
     id: 'user.profile.auditList.usernameRequested'
+  },
+
+  sendUserNameReminder: {
+    defaultMessage: 'Sent Username Reminder',
+    description: 'Description for send username reminder audit action',
+    id: 'user.profile.auditList.sentUsernameReminder'
```

Comment:
@favour234 translations are required in Farajaland

Reply:
alright